python/spark_package/spark_package.py

```diff
@@ -446,8 +449,7 @@ def validate_and_return_sp_dep(line):
 
 
 def pom_pretty_print(f):
-    return '\n'.join([line for line in dom.parseString(f)
-        .toprettyxml(indent=' ' * 2, encoding='UTF-8').split('\n') if line.strip()])
+    return dom.parseString(f).toprettyxml(indent=' ' * 2, encoding='UTF-8').decode("utf-8").strip('\n')
```

Comment:
All right. Actually, the result of Python3 is difference from that of Python2.

#### Python3

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.
apache.org/xsd/maven-4.0.0.xsd">
  <groupId>yu-iskw</groupId>
  <artifactId>bisecting-kmeans</artifactId>
  <version>0.1</version>
  <repositories>
    <repository>
      <id>SparkPackagesRepo</id>
      <name>Spark Packages Repository</name>
      <url>http://dl.bintray.com/spark-packages/maven/</url>
      <layout>default</layout>
    </repository>
  </repositories>
</project>
```

#### Python2

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <groupId>
    yu-iskw
  </groupId>
  <artifactId>
    bisecting-kmeans
  </artifactId>
  <version>
    0.1
  </version>
  <repositories>
    <repository>
      <id>
        SparkPackagesRepo
      </id>
      <name>
        Spark Packages Repository
      </name>
      <url>
        http://dl.bintray.com/spark-packages/maven/
      </url>
      <layout>
        default
      </layout>
    </repository>
  </repositories>
</project>
```